### PR TITLE
Add style linter

### PR DESCRIPTION
**Pull request by maal26**
related #9 

This PR will add some config files to guarantee that all developers follow the same code style guide :shipit:

The config files are:

- .editorconfig
- .eslintrc.json

### CODE REVIEW

**Comment by migo315** [comment]
Thanks for your pull request. This PR has merge conflicts - can you resolve them? I will merge it after that. ;-)

**Comment by maal26** [comment]
Ok, it's done. Let me know if you need anything changed.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
```
Comment (philipmckenna) [inline .editorconfig:6]: NAB: Just a opinion that should be open to discussion with the owner, airbnb-base uses indent_size = 2.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "es6": true
+    },
+    "extends": ["eslint:recommended"],
```
Comment (philipmckenna) [inline .eslintrc.json:5]: NAB: should consider and discuss other extensions, personally I use airbnb-base and prettier which are very opinionated.

**Review by migo315 — APPROVED** [review]
Great! Thanks ;-)

**Review by philipmckenna — COMMENTED** [review]
Hey 👋 just hoping to get a few discussions going on this as its quite a heavily opinionated topic.

File: .editorconfig (hunk 1 above)
Comment (migo315) [reply .editorconfig:6]: My current IDE settings using 4 by default for javascript. So, that's ok for me.

File: .eslintrc.json (hunk 2 above)
Comment (migo315) [reply .eslintrc.json:5]: I work more with PHP than with Javascript. Therefore, I do not know the most extensions and standards. You are free to create a pull request if you can optimize something. ;-)

**Pull request merged**